central/graphql/resolvers/plotted_node_vulnerabilities.go

```diff
@@ -24,8 +26,32 @@ type PlottedNodeVulnerabilitiesResolver struct {
 }
 
 func newPlottedNodeVulnerabilitiesResolver(ctx context.Context, root *Resolver, args RawQuery) (*PlottedNodeVulnerabilitiesResolver, error) {
-	query := withNodeCveTypeFiltering(args.String())
-	allCveIds, fixableCount, err := getPlottedVulnsIdsAndFixableCount(ctx, root, RawQuery{Query: &query})
+	if !features.PostgresDatastore.Enabled() {
+		q := withNodeCveTypeFiltering(args.String())
+		allCveIds, fixableCount, err := getPlottedVulnsIdsAndFixableCount(ctx, root, RawQuery{Query: &q})
+		if err != nil {
+			return nil, err
+		}
+
+		return &PlottedNodeVulnerabilitiesResolver{
+			root:    root,
+			all:     allCveIds,
+			fixable: fixableCount,
+		}, nil
+	}
+
+	query, err := getPlottedVulnsV1Query(args)
+	if err != nil {
+		return nil, err
+	}
+	all, err := root.NodeCVEDataStore.Search(ctx, query)
+	if err != nil {
+		return nil, err
+	}
+	allCveIds := search.ResultsToIDs(all)
+
+	fixableCount, err := root.NodeCVEDataStore.Count(ctx,
+		search.ConjunctionQuery(query, search.NewQueryBuilder().AddBools(search.Fixable, true).ProtoQuery()))
```

Comment:
Ensure that the incoming query does not have `Fixable` in it else the result could be unexpected.

Reply:
Hmm, I see a few other places where a conjunction query is added without ensuring the input query does not have the fieldLabel. I would add `ExcludeFieldLabel` options to other places as well.